src/resources/views/crud/fields/relationship.blade.php

```diff
@@ -40,10 +40,15 @@
             }
 
 
-            // the dev is trying to create a field for the ENTIRE hasOne/morphOne relationship
-            abort(500, "<strong>The relationship field does not support <code>{$field['relation_type']}</code> that way.</strong><br> Please change your <code>{$field['name']}</code> field so that its <i>name</i> also includes the editable attribute on the related model, using dot notation (eg. <code>address.postal_code</code>). If you need to edit more attributes, add a new field for each one (eg. <code>address.postal_code</code>). See <a target='_blank' href='https://backpackforlaravel.com/docs/crud-fields#hasone-1-1-relationship'>the docs</a> for more information.");
-            // TODO: if "fields" is not defined, tell the dev to define it (+ link to docs)
-            // TODO: if "fields" is defined, load a repeatable field with one entry (and 1 entry max)
+            // -----
```

Comment:
Heads-up @pxpm , I moved this a little bit lower, to remove the TODOs, and to echo a message if no `fields` are provided. As I said, we should make the fields as helpful as possible. If we know `fields` are missing, that's what we tell the dev.